geniza/footnotes/migrations/0001_initial.py

```diff
@@ -1,21 +1,29 @@
-# Generated by Django 3.1.7 on 2021-04-01 22:12
+# Generated by Django 3.1.6 on 2021-04-07 13:20
```

Comment:
why'd the minor version go down? that seems odd

Reply:
😆 wow, eagle eyes. Kevin must have a newer version installed than I do!